runtime/default.policy

```diff
@@ -94,13 +94,17 @@ rules:
       open.filename =~ "/etc/ssl/certs/*" && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
     tags:
       technique: T1338
-  - id: pci_11_5
+  - id: pci_11_5_bin
     description: Modification of critical system files
     expression: >-
       (open.filename =~ "/bin/*" || 
       open.filename =~ "/sbin/*" || 
       open.filename =~ "/usr/bin/*" || 
-      open.filename =~ "/usr/sbin/*" || 
-      open.filename =~ "/opt/*") && 
+      open.filename =~ "/usr/sbin/*") &&
+      open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+  - id: pci_11_5_opt
```

Comment:
I think, that it could be renamed as pci_11_5_critical_files.

Reply:
👍